website/docs/concepts/recorder_service/README.md

```diff
@@ -61,24 +83,28 @@ used for depth perception and 3D mapping.
 - `oak/imu`: Streams 3-D accelerometer & gyroscope values from the Oak camera's internal IMU,
 giving insights into the robot's orientation and movement.
 - `oak/left`: Outputs the feed from the stereo left camera of the Oak device.
-- `oak/right`: Outputs the feed from the stereo right camera of the Oak device.
 - `oak/rgb`: Streams the stereo RGB image from the Oak device, providing full-color
 visuals of the robot's environment.
+- `oak/right`: Outputs the feed from the stereo right camera of the Oak device.
 
-### GPS Topics
+### System Monitor Topics
 
-- `gps/health`: Offers health metrics for the GPS service, such as publishing frequency.
-- `gps/pvt`: Streams GPS Position, Velocity, and Time (PVT) messages, giving a comprehensive
-view of the robot's location and movement.
-- `gps/relposned`: Provides GPS relative position in North, East, Down (N-E-D) coordinates,
-offering a detailed spatial orientation of the robot.
+- `system_monitor/health`: Delivers health metrics for the system_monitor service,
+including publishing frequency.
+- `system_monitor/auto_mode`: Streams auto mode state (either true or false).
+- `system_monitor/batteries_state`: Streams the percent of battery left.
+- `system_monitor/cpu_usage`: Streams the percent of CPU currently being consumed on the brain.
+- `system_monitor/filter_state`: Streams the filter service state.
+- `system_monitor/wifi_state`: Streams auto mode state (either true or false).
+- `system_monitor/virtual_memory_usage`: Streams athe percent of virtual memory being consumed
+by the brain.
 
-### Filter Topics
+### Track Follower Topics
 
-- `filter/health`: Delivers health metrics for the Filter service, including publishing frequency.
-- `filter/state`: Streams the filter state, which combines the robot's pose, orientation,
-filter state uncertainties, and filter convergence status.
-This topic is essential for understanding the robot's current state and its accuracy.
+- `track_follower/health`: Delivers health metrics for the track_follower service,
+including publishing frequency.
+
+- `track_follower/state`: Streams the current state of a track being followed.
```

Comment:
Listing out all the topics of the other services feel redundant. These should be listed out on the individual service pages and we link to there. We do not want to have to maintain the list and description of all topics in multiple places

Reply:
That makes sense. I added links for can, filter, gps, oak, and track_follower (15d1c49550da86d19e07ff5ed1274ad4c3d37903). But I left the streams for system_monitor as is since we don't have a dedicated page for it.